server/cli/start.go

```diff
@@ -41,21 +40,21 @@ var Context = server.NewContext()
 
 // An initCmd command initializes a new Cockroach cluster.
 var initCmd = &commander.Command{
-	UsageLine: "init <storage-location>",
+	UsageLine: "init -stores=<storage-location>[,<storage-location>...]",
 	Short:     "init new Cockroach cluster and start server",
 	Long: `
-Initialize a new Cockroach cluster using the first argument as the
-storage location used to bootstrap the first replica of the first
-range. If the storage location is already part of a pre-existing
-cluster, the bootstrap will fail.
+Initialize a new Cockroach cluster using the first argument as one or more 
```

Comment:
Not "first argument", instead use: "using the -stores command line flag to specify one or more storage locations".